tests/plugins.ts

```diff
@@ -0,0 +1,16 @@
+//  a plugin example with the protractor plugin interface
+import { ProtractorPlugin } from 'protractor';
+
```

Comment:
Why do we need this `Plugin.ts` file?

Reply:
Good find, I removed the usage of this but aparently oversight the actual implementation.